src/kudu/consensus/log_index.cc

```diff
@@ -168,11 +199,17 @@ string LogIndex::GetChunkPath(int64_t chunk_idx) {
   return StringPrintf("%s/index.%09" PRId64, base_dir_.c_str(), chunk_idx);
 }
 
-Status LogIndex::OpenAllChunksOnStartup(Env *env) {
+Status LogIndex::OpenAllChunksOnStartup(
+    Env *env,
+    const scoped_refptr<MetricEntity>& metric_entity) {
   DCHECK(env);
   std::vector<std::string> children;
   RETURN_NOT_OK(env->GetChildren(base_dir_, &children));
 
```

Comment:
I am not sure children is sorted in alphanumeric order when we get from env

Reply:
`children` need not be sorted for the changes in this PR. `open_chunks_` need to be sorted for the logic of mmapping the highest numbered `chunk_idx` and it is guaranteed to be sorted since it is a `std::map`